### PR TITLE
Fixes #462: Bugged links and Irrelevant content

### LINKED ISSUE #462: Bugged links and irrelevant content
### Expected results
Working links with relevant content
### Actual results
Bugged links and content from template

## Would you like to work on this
Yeah sure!

### PR DESCRIPTION
**Pull request by saarthakchats**
Fixes #462

### CODE REVIEW

**Comment by saarthakchats** [comment]
Issue #462

File: index.html [hunk 1]
```diff
@@ -538,7 +538,7 @@ <h3 class="mb40 mb-xs-24">FOSSASIA Google Code-In 2016/17</h3>
 
 			</div>
 			<div class="align-bottom text-center">
-				<a class="btn btn-white mb32" href="#">Connect</a>
+				<a class="btn btn-white mb32" href="2018.fossasia.org"></a>
```
Comment (harshit98) [inline index.html]: There is nothing between anchor tags. Why you've removed `Connect` ? Just replace `href="#"` to `href="2018.fossasia.org"`.

**Comment by saarthakchats** [comment]
@harshit98 Is it fine now :(

**Comment by saarthakchats** [comment]
https://saarthakchats.github.io/l-l-h-gci17.fossasia.org/

File: index.html [hunk 2]
```diff
@@ -550,36 +550,27 @@ <h3 class="mb40 mb-xs-24">FOSSASIA Google Code-In 2016/17</h3>
...
+                               </a>
+                    </li>
+                    <li>
+                            <a href="https://flickr.com/photos/fossasia">
+                                    <i class="ti-flickr-alt" aria-hidden="true"></i>
+                            </a>
+                    </li>
+                    <li>
+                            <a href="https://github.com/fossasia">
+									<i class="ti-github icon-i" aria-hidden="true"></i>
```
Comment (harshit98) [inline index.html]: Indent this line properly.

Comment (harshit98) [reply index.html]: From line 571 to 572.

**Comment by saarthakchats** [comment]
@harshit98 Done!

**Review by harshit98 — APPROVED** [review]

**Comment by saarthakchats** [comment]
Pls merge this. Thanks

**Comment by saarthakchats** [comment]
Thanks @harshit98

**Comment by saarthakchats** [comment]
https://codein.withgoogle.com/dashboard/task-instances/5609258374660096/
Pls approve thanks

**Pull request merged**